src/lib.rs

```diff
@@ -436,6 +465,51 @@ impl Entry {
         biblatex
     }
 
+    /// Serialize this entry into a BibLaTeX string.
+    ///
+    /// Same as `to_biblatex_string`. Takes an extra parameter `order` which is a &vec of &str.
+    /// Will order the entry fields by placing the fields in the vec first (in that order).
+    /// The remaining ones will be ordered alphabetically
+    ///
+    /// e.g. if `order` is `vec!["title", "author"]`, will place the author and title fields first
+    /// and then the rest of the fields ordered alphabetically
+    pub fn to_biblatex_string_ordered(&self, order: &Vec<&str>) -> String {
+        let mut biblatex = String::new();
+        let ty = self.entry_type.to_biblatex();
+
+        writeln!(biblatex, "@{}{{{},", ty, self.key).unwrap();
+
+        for (key, value) in self.fields.iter().sorted_by_key(|x| {
+            for (idx, ord_param) in order.iter().enumerate() {
+                // If the key coincides with the nth element of the order vector
+                if x.0 == ord_param {
+                    // If order is ["title", "author"], author key will be ordered as "001".
+                    return format!("{:03}", idx);
+                }
+            }
+            // If not in the order vec, order as the key
+            // (with the modifications that will take place later on in the function)
+            match x.0.as_ref() {
+                "journal" => "journaltitle".to_string(),
+                "address" => "location".to_string(),
+                "school" => "institution".to_string(),
+                k => k.to_string(),
+            }
+        }) {
+            let key = match key.as_ref() {
+                "journal" => "journaltitle",
+                "address" => "location",
+                "school" => "institution",
+                k => k,
+            };
```

Comment:
Normalizing the order may be a good idea but I'm not a fan of normalizing the field names as well, especially since it is stated nowhere in the documentation comments.